config/config.go

```diff
@@ -16,6 +16,7 @@ type Config struct {
 	HealthCheckCriticalTimeout time.Duration `envconfig:"HEALTHCHECK_CRITICAL_TIMEOUT"`
 	OutputFilePath             string        `envconfig:"OUTPUT_FILE_PATH"`
 	KafkaConfig                KafkaConfig
+	ContentUpdatedTopicFlag    bool `envconfig:"CONTENT_UPDATED_TOPIC_FLAG"`
```

Comment:
Can you format this please? You may need to run `make fmt`

Reply:
Please can you show me where the ContentUpdatedTopicFlag is being used? I might be missing something but I can't see it being used anywhere so I don't know how to test it.

Reply:
I have updated the PR including READMe to use this new topic, pls revisit this and let me know if you need further clarifications.